regress/regress_test.go

```diff
@@ -17,6 +17,7 @@ func TestRegress(t *testing.T) {
 		t.Fatal(err)
 	}
 	for _, test := range tests {
+		test := test
```

Comment:
Oh no.

Reply:
"Oh no" that it's been broken-ish all this time?

Reply:
Yes.

Reply:
Yeah; `t.Parallel` is an interesting beast.